src/mmap_unix.rs

```diff
@@ -24,10 +24,15 @@ use crate::mmap::{check_file_offset, AsSlice};
 use crate::volatile_memory::{self, compute_offset, VolatileMemory, VolatileSlice};
 
 /// Error conditions that may arise when creating a new `MmapRegion` object.
+#[allow(missing_docs)]
```

Comment:
That's to avoid a complain for the anonymous structs added to `Error` for the `InvalidPointer` and `InvalidSize` variants. We already have something similar in `volatile_memory::Error`.

On the other hand, I've added that just for the coverage test. If we remove that test, we can get rid of this.

Reply:
That'd be nice.